components/actionbar/index.css

```diff
@@ -12,42 +12,31 @@
  */
 
 .spectrum-ActionBar {
-	--spectrum-actionbar-popover-background-color: var(--spectrum-gray-25);
-	--spectrum-actionbar-popover-border-color: var(--spectrum-gray-400);
-
+	/* Layout */
 	--spectrum-actionbar-height: var(--spectrum-action-bar-height);
-	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-100);
+	--spectrum-actionbar-minimum-width: var(--spectrum-action-bar-minimum-width);
+	--spectrum-actionbar-corner-radius: var(--spectrum-corner-radius-medium-size-extra-large);
+	--spectrum-actionbar-spacing-label-to-action-group: var(--spectrum-action-bar-label-to-action-group-area);
+	--spectrum-actionbar-spacing-action-group-edge: var(--spectrum-action-bar-edge-to-content-area);
+	--spectrum-actionbar-close-button-to-counter: var(--spectrum-action-bar-close-button-to-counter);
 
 	/* item counter field label */
 	--spectrum-actionbar-item-counter-font-size: var(--spectrum-font-size-100);
```

Comment:
Should we update this to use the token that's in the design? This isn't something you introduced, but the token specs call for `action-bar-counter-font-size`, so should we use that instead of just `font-size-100`? (that's what the counter-font-size token is set to already, but would design be looking for counter-font-size when they review? 🤷‍♀️ )